packages/scrum-server/src/config/corsOptions.js

```diff
@@ -0,0 +1,17 @@
+const keys = require('./keys');
+
+const corsWhiteListUrl = process.env.CORS_WHITELIST_URL.split(' ');
```

Comment:
Can we get it added to the keys and then read it from there? that will give us also default values

Reply:
Done